### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by kunsungwoo** [THU-MIG/yolov10#268]
[1]
results= model.train(data='/content/sample_data/test/data.yaml',epochs=8,imgsz=640,batch=16,patience=3,save_period=1,name='yolo8',lr0=0.001,lrf=0.0001,exist_ok=True,pretrained=True ,translate=0.02,scale=0.02,mosaic=0.01,mixup=0.01,box=20,close_mosaic=3,shear=0.0,erasing=0,perspective=0.0,flipud=0,fliplr=0,iou=0.95,copy_paste=0,optimizer='AdamW',cos_lr=True) #nn3

[2]
results= model.train(data='/content/sample_data/test/data.yaml',epochs=8,imgsz=640,batch=16,patience=3,save_period=1,name='yolo11',lr0=0.001,lrf=0.0001,exist_ok=True,pretrained=True ,translate=0.02,scale=0.02,mosaic=0.01,mixup=0.01,box=20,close_mosaic=3,shear=0.0,erasing=0,perspective=0.0,flipud=0,fliplr=0,iou=0.95,copy_paste=0,optimizer='AdamW',cos_lr=True) #nn3

There were no problems when learning with [1]. However, when learning with [2], the loss value appears as nan starting from epoch6. All parameters are the same and only some learning data has been added. Why does this problem occur?
